docs/UserGuide.md

```diff
@@ -216,15 +216,15 @@ Updates the status of a property from Option to Purchase, to Sales and Purchase
 
 Formats:
 * `update INDEX u/new AMOUNT`
-* `update INDEX u/[proceed][cancel]`
+* `update INDEX u/proceed`
+* `update INDEX u/cancel`
```

Comment:
Actually will it be better if you have only 1 format, `update INDEX u/OPTION`, then you specify what the options are? Similar to how `find` is structured?

Not really sure whats the best way to present this...

Reply:
i tried my best to match the find format!